lua/core/mappings.lua

```diff
@@ -252,6 +254,7 @@ if is_available "toggleterm.nvim" then
 end
 
 if is_available "nvim-dap" then
+  maps.n["<leader>D"] = sections.D
```

Comment:
would it be possible to make these lowercase by default? discussed previously, I think we said we would do it in v3

PS: Happy New Year

Reply:
I'm not planning on changing this since `<leader>d` is a good binding for the dashboard and we have function key shortcuts for easy access for this functionality

Reply:
Hm we aren't binding `<leader>h` anymore, maybe we can make that like "Home" and do `<leader>d` for debugging

Reply:
Resolved in latest commits